builder/bootloader.py

```diff
@@ -78,16 +118,20 @@ def get_suitable_optiboot_binary(framework_dir, board_config):
 
 fuses_action = env.SConscript("fuses.py", exports="env")
 
-lock_bits = board.get("bootloader.lock_bits", "0x0F")
-unlock_bits = board.get("bootloader.unlock_bits", "0x3F")
+if core == "MicroCore":
+    lock_bits = board.get("bootloader.lock_bits", "0xFF")
+    unlock_bits = board.get("bootloader.unlock_bits", "0xFF")
+else:
+    lock_bits = board.get("bootloader.lock_bits", "0x0F")
+    unlock_bits = board.get("bootloader.unlock_bits", "0x3F")
 
 env.Replace(
     BOOTUPLOADER="avrdude",
     BOOTUPLOADERFLAGS=[
         "-p",
         "$BOARD_MCU",
         "-C",
-        '"%s"'
+        "%s"
```

Comment:
It would be great to separate concerns regarding added functionality. Do you mind moving this changes to a separate PR?

Reply:
Sure! I'll remove the commit, rebase and submit a new PR when this PR is ready to be merged